docs/content/Contribution guidelines/operations/code-management.md

```diff
@@ -7,3 +7,128 @@
 <!-- Code management
   Prow, Gh actions broken links, pr verifier, emoji in titles of prs, add issue to project. Add pr to project. Check spelling errors, wordlist.txt, 
 Quay.io -->
+# Code Management
+
+## Fork kubestellar into your own repo, create a local branch, set upstream to kubestellar, add and commit changes to local branch, and squash your commits
```

Comment:
You might want to split this in two, the current title combines setup done once with stuff repeated per PR

Reply:
Thanks for the comments @MikeSpreitzer. Good point, no need to fork again.